OsmAnd/res/values/strings.xml

```diff
@@ -41,23 +40,23 @@
     <string name="pseudo_mercator_projection">Pseudo-Mercator projection</string>
     <string name="keep_screen_off">Keep screen off</string>
     <string name="keep_screen_on">Keep screen on</string>
-    <string name="turn_screen_on_wake_time_descr">Select the screen timeout after waking up. (\"%1$s\" applies no timeout.)</string>
+    <string name="turn_screen_on_wake_time_descr">Select how long the screen should be kept on. (\"%1$s\" keeps the screen from turning off.)</string>
```

Comment:
It's tricky, this timeout is not the regular timeout, i really only applies after a wake-up.